### PR TITLE
Add new and update status / success icons

### PR DESCRIPTION
**Pull request by origami-z**
- [x] Add / update new icons
- [x] Deprecate icons (modify scripts)
-  ~~Relink status indicator icons in core~~

Updated icon generation script to make it pass Biome organize imports lint error.

~~Updated some cypress test for status indicator, so they are not longer testing test-id (which would be better captured via visual regression), updated the test to label / role to reflect non-visual aspects ~~

#4100, #4347

### CODE REVIEW

**Comment by origami-z** [comment]
Ready for review. Chromatic snapshot diff can be seen here: https://www.chromatic.com/build?appId=628e27029fcf95004a62cafd&number=17542

File: packages/core/src/__tests__/__e2e__/banner/Banner.cy.tsx [hunk 1]
```diff
@@ -13,10 +13,10 @@ describe("GIVEN a Banner", () => {
   it("THEN should render status", () => {
     cy.mount(<StatusesPrimary />);
 
-    cy.findByTestId("InfoSolidIcon").should("exist");
-    cy.findByTestId("SuccessTickIcon").should("exist");
-    cy.findByTestId("WarningSolidIcon").should("exist");
-    cy.findByTestId("ErrorSolidIcon").should("exist");
+    cy.findByRole("img", { name: "info" }).should("exist");
```
Comment (joshwooding) [inline packages/core/src/__tests__/__e2e__/banner/Banner.cy.tsx:16]: I'm not against the test changes but it's worth highlighting these don't test the same thing, one is testing the actual icon and the other is testing the accessible name of the icon. Which should be fine paired with our visual regression testing.

File: packages/icons/scripts/generateIcons.mjs [hunk 2]
```diff
@@ -37,6 +41,38 @@ function pascalCase(str) {
 
   return capital.join("");
 }
+
+/**
+ * Turn `AppleBanana` into `Apple Banana`
+ * @param {string} str
+ **/
+function breakParcalCasingWithSpace(str) {
```
Comment (mark-tate) [inline packages/icons/scripts/generateIcons.mjs:49]: did we mean Pascal ?

File: packages/icons/scripts/generateIcons.mjs [hunk 3]
```diff
@@ -197,6 +243,11 @@ const generateIconComponents = async ({
         componentName,
         ariaLabel: iconTitle,
         viewBox: viewBox ?? "0 0 12 12",
+        // note: triple mustache is used here for unescapted version
```
Comment (mark-tate) [inline packages/icons/scripts/generateIcons.mjs:246]: typo unescapted

**Comment by bhoppers2008** [comment]
What's the adoption path for the success, status and checkmark changes. I think we discussed:

**Success updates**
No immediate change to components
Existing icons remain but deprecated
New icons are added
Update in components comes later
(Noticeable visual change)

**Progress tracker**
Success progress icon removed (or deprecated?... what is necessary, they are essentially the same)
New success icon used immediately
(Very limited visual change)

**Checkmark icon**
Checkmark icons added
Used in checkbox component and grid theme immediately
(Considered a fix–enables a single icon used across all densities)
Previous checkbox tick icons removed/deprecated(??)

**Comment by origami-z** [comment]
> **Success updates**
> No immediate change to components
> Existing icons remain but deprecated
> New icons are added
> Update in components comes later
> (Noticeable visual change)

This PR only covers success icon. In Figma, icon changes are rolled out in components? @pseys

I'm can take out component changes (around `StatusIndicator`) in code

**Review by joshwooding — APPROVED** [review]

**Review by Fercas123 — APPROVED** [review]

**Pull request merged**